binding.gyp

```diff
@@ -6,11 +6,12 @@
     {
       "target_name": "protagonist",
       "include_dirs": [
-        "./snowcrash/src",
-        "./snowcrash/ext/markdown-parser/src",
-        "./snowcrash/ext/markdown-parser/ext/sundown/src",
-        "./snowcrash/ext/markdown-parser/ext/sundown/html",
-        "./snowcrash/ext/sos/src",
+        "drafter/src",
```

Comment:
Please do not do the back port until after the sync today. I need to discuss a few things with the team.